npe2/manifest/schema.py

```diff
@@ -53,20 +48,17 @@ class DiscoverResults(NamedTuple):
 
 class PluginManifest(BaseModel):
 
-    # VS Code uses <publisher>.<name> as a unique ID for the extension
-    # should this just be the package name ... not the module name? (probably yes)
-    # do we normalize this? (i.e. underscores / dashes ?)
-    # TODO: enforce that this matches the package name
     name: str = Field(
         ...,
-        description="The name of the plugin - should be all lowercase with no spaces.",
+        description="The name of the plugin. Should correspond to the python "
+        "package name for this plugin.",
```

Comment:
This is related to the discussion in #37. I think the best approach is to require the name to be the package name. I haven't changed any of the validation so #37 is still an issue.